packages/node/src/login/oidc/oidcHandlers/ClientCredentialsOidcHandler.ts

```diff
@@ -59,7 +60,6 @@ export default class ClientCredentialsOidcHandler implements IOidcHandler {
       // If a redirect URL is present, the static client should use the
       // authorization code flow.
       typeof oidcLoginOptions.redirectUrl !== "string" &&
-      typeof oidcLoginOptions.client.clientId === "string" &&
```

Comment:
Why has this line been removed?

Reply:
I should have added a comment in GitHub. It is redundant as all variants of IClient have clientId as a mandatory string. My IDE pointed it out but I checked myself by looking at the types and agreed.